Dockerfile

```diff
@@ -1,7 +1,8 @@
-ARG GO_VERSION=1.21-bullseye
+ARG GO_VERSION=1.21-bookworm
 ARG IMAGE_TYPE=extras
 # extras or core
 
+FROM rocm/dev-ubuntu-22.04:6.0-complete as rocm
```

Comment:
maybe we can have rocm images like we have for cublas? so we have lighter images for AMD

Reply:
There are a bunch of limitations currently with how AMD is packaging ROCm. I will see if I can come up with a good way to split it up

Reply:
sounds good, also have a look at https://github.com/mudler/LocalAI/pull/1660 as is quite similar

Reply:
ROCm only installs currently on CentOS and Ubuntu, it has a glibc dependency pinned to the Ubuntu lunar glibc (2.36). There are no versions of the go builder that have that version. Bullseye is too old, Bookworm is too new. So I'm having to do this awkward side-loading from another container.

I am trying something else right now